src/Fabulous.Core/Fabulous.Core.fsproj

```diff
@@ -10,8 +10,8 @@
     <ItemGroup>
         <Compile Include="ViewElement.fs" />
         <Compile Include="ViewConverters.fs" />
-        <Compile Include="Xamarin.Forms.Core.fs" />
         <Compile Include="ViewHelpers.fs" />
```

Comment:
I moved `ViewHelpers.fs` to remove the dependency it had on `Xamarin.Forms.Core.fs` before.